server.js

```diff
@@ -124,7 +139,7 @@ app.get('/chart.png', (request, response, callback) => {
 app.listen(port, () => {
   console.log(`Listening on http://0.0.0.0:${port}`);
   if (process.env.NODE_ENV !== 'production') {
-    console.info('Try it out: \x1b[33;1mhttp://0.0.0.0:2197/chart.png?width=750&height=600&columns=2015-05-09,2015-06-09,2015-07-09,2015-08-09,2015-09-09,2015-10-09,2015-11-09,2015-12-09,2016-01-09,2016-02-09,2016-03-09&data=438500,522500,339250,289000,384750,289625,226250,475000,348500,279900,170000|208000,215000,218000,215000,212500,216958,215000,215000,215000,218000,216312&line_colors=2ba8de,9ba1a6&legend_labels=Your+Home,Phoenix\x1b[0m');
+    console.info('Try it out: \x1b[33;1mhttp://0.0.0.0:2197/chart.png?width=750&height=600&columns=2015-05-09,2015-06-09,2015-07-09,2015-08-09,2015-09-09,2015-10-09,2015-11-09,2015-12-09,2016-01-09,2016-02-09,2016-03-09&data=438500,522500,339250,289000,384750,289625,226250,475000,348500,279900,170000|208000,215000,218000,215000,212500,216958,215000,215000,215000,218000,216312&line_colors=1C85E8,D0D3E0&legend_labels=Your+Home,Phoenix\x1b[0m');
```

Comment:
if you need different colors to be on-brand, you'll need to also coordinate a server-side change to the emails since this won't change what's requested there.